pkg/template/registry/rest.go

```diff
@@ -120,5 +120,5 @@ func (s *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 	if s.legacyReturn {
 		return cfg, nil
 	}
-	return &kapi.List{Items: cfg.Items}, nil
+	return tpl, nil
```

Comment:
Fix label application (moved it into Process where it belongs) and added tests (so we actually, you know, know that it works).

----- Original Message -----
> If I did.... we didn't have any test for it.
> 
> ----- Original Message -----
> > > @@ -120,5 +120,5 @@ func (s *REST) Create(ctx kapi.Context, obj
> > > runtime.Object) (runtime.Object, err
> > >  	if s.legacyReturn {
> > >  		return cfg, nil
> > >  	}
> > > -	return &kapi.List{Items: cfg.Items}, nil
> > > +	return tpl, nil
> > 
> > haven't you lost the label application aspect of template processing here?
> > 
> > 
> > ---
> > Reply to this email directly or view it on GitHub:
> > https://github.com/openshift/origin/pull/1979/files#r29373631
>